bokeh_app/make_settings/main.py

```diff
@@ -16,101 +19,123 @@
 # Bokeh imports
 from bokeh.io import curdoc
 from bokeh.layouts import column, row, grid
-from bokeh.models import Div, CheckboxGroup, DatePicker, RadioButtonGroup, MultiChoice
+from bokeh.models import Div, CheckboxGroup, DatePicker, RadioButtonGroup
+from bokeh.models import MultiChoice, Panel, Tabs, MultiSelect, CustomJS
 from bokeh.models.widgets import TextInput, Button, Paragraph
+from bokeh.events import ButtonClick

```

Comment:
I tested the code in docker and faced 2 issues, which are not relevant outside of docker, but it's good to mention now.

First, `USER` env var is not set, which is required by `create_newcase` (I'm not sure why. This should be available by default). This can be fixed by importing `os` and adding the following line after the imports: `os.environ['USER'] = subprocess.run(['whoami'], stdout=PIPE).stdout.decode('utf-8').strip()`.

The second issue is about `data` folder permissions inside docker, though we can ignore this for now. I think it's easier to merge this first and then fix this issue so we can test everything together with the latest docker changes.

Reply:
Thanks, Kaveh! I'll merge it for now then so that everyone can easily test the new adaptations. There is also one UX issue with executing multiple call-backs in the bokeh up simultaneously, I will show what I mean during our meeting and open a GH issue later.